ragstack/colbert/cassandra_retriever.py

```diff
@@ -113,9 +115,12 @@ def retrieve(
         doc_futures = []
         for qv in query_encodings:
             # per token based retrieval
-            doc_future = self.vector_store.session.execute_async(
-                self.vector_store.query_colbert_ann_stmt, [list(qv), top_k]
-            )
+            bound_statement = self.vector_store.query_colbert_ann_stmt.bind([list(qv), top_k])
+
+            # top-k queries can only be run with consistency level ONE / LOCAL_ONE / NODE_LOCAL
```

Comment:
I see Nicolo's comment on this might be Astra only. Can you provide an option to allow user to suppress the warning as needed? 

LGTM